backend/internal/server/handlers_case_batch_test.go

```diff
@@ -0,0 +1,460 @@
+				"pipeline_version": "1.0.0",
+				"output_documents": [{
+					"data_category_code": "genomic",
+					"data_type_code": "aligned_reads",
+					"format_code": "bam",
+					"name": "output.bam",
+					"size": 1000000,
+					"url": "https://example.com/output.bam"
+				}]
+			}]
+		}]
+	}`
+	req, _ := http.NewRequest(http.MethodPost, "/cases/batch", bytes.NewBuffer([]byte(body)))
+	req.Header.Set("Content-Type", "application/json")
+	w := httptest.NewRecorder()
+	router.ServeHTTP(w, req)
+
+	assert.Equal(t, http.StatusBadRequest, w.Code)
+}
+
+func TestPostCaseBatchHandler_MissingPatients(t *testing.T) {
+	gin.SetMode(gin.TestMode)
+	repo := &MockBatchRepository{}
+	auth := &testutils.MockAuth{}
+
+	router := gin.Default()
+	router.POST("/cases/batch", PostCaseBatchHandler(repo, auth))
+	body := `{
+		"cases": [{
+			"submitter_case_id": "case1",
+			"type": "germline",
+			"status_code": "active",
+			"project_code": "proj1",
+			"category_code": "postnatal",
+			"sequencing_experiments": [{
+				"aliquot": "alq1",
+				"sample_organization_code": "org1",
+				"submitter_sample_id": "s1"
+			}],
+			"tasks": [{
+				"type_code": "alignment",
+				"pipeline_version": "1.0.0",
+				"output_documents": [{
+					"data_category_code": "genomic",
+					"data_type_code": "aligned_reads",
+					"format_code": "bam",
+					"name": "output.bam",
+					"size": 1000000,
+					"url": "https://example.com/output.bam"
+				}]
+			}]
+		}]
+	}`
+	req, _ := http.NewRequest(http.MethodPost, "/cases/batch", bytes.NewBuffer([]byte(body)))
+	req.Header.Set("Content-Type", "application/json")
+	w := httptest.NewRecorder()
+	router.ServeHTTP(w, req)
+
+	assert.Equal(t, http.StatusBadRequest, w.Code)
+}
+
+func TestPostCaseBatchHandler_EmptyPatients(t *testing.T) {
+	gin.SetMode(gin.TestMode)
+	repo := &MockBatchRepository{}
+	auth := &testutils.MockAuth{}
+
+	router := gin.Default()
+	router.POST("/cases/batch", PostCaseBatchHandler(repo, auth))
+	body := `{
+		"cases": [{
+			"submitter_case_id": "case1",
+			"type": "germline",
+			"status_code": "active",
+			"project_code": "proj1",
+			"category_code": "postnatal",
+			"patients": [],
+			"sequencing_experiments": [{
+				"aliquot": "alq1",
+				"sample_organization_code": "org1",
+				"submitter_sample_id": "s1"
+			}],
+			"tasks": [{
+				"type_code": "alignment",
+				"pipeline_version": "1.0.0",
+				"output_documents": [{
+					"data_category_code": "genomic",
+					"data_type_code": "aligned_reads",
+					"format_code": "bam",
+					"name": "output.bam",
+					"size": 1000000,
+					"url": "https://example.com/output.bam"
+				}]
+			}]
+		}]
+	}`
+	req, _ := http.NewRequest(http.MethodPost, "/cases/batch", bytes.NewBuffer([]byte(body)))
+	req.Header.Set("Content-Type", "application/json")
+	w := httptest.NewRecorder()
+	router.ServeHTTP(w, req)
+
+	assert.Equal(t, http.StatusBadRequest, w.Code)
+}
+
+func TestPostCaseBatchHandler_MissingSequencingExperiments(t *testing.T) {
```

Comment:
I think it's a use case to be able to create a case before sequencing (so no sequencing and no task when the case is created)
@jecos  am I wrong ?

Reply:
I made `sequencing_experiments` optional for now and commented tasks

Reply:
But we can keep it mandatory for now, because we dont have an endpoint to patch a case for now. When we'll support this use-case, we would change sequencing_experiments to be optional

Reply:
Ok I set it back to sequencing experiments being mandatory